Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -3,7 +3,10 @@
 
 #pragma once
 
+#ifdef WIN32
```

Comment:
i dont think `ifdef WIN32` is required here?

Reply:
I don't think it's required to build on every platform, but all of my changes were in DInput which is Direct Input. Direct Input is a Windows exclusive input mode so I decided not to show the UI elements on other platforms since those UI elements won't do anything.

Reply:
I didn't think so either, but I was getting build errors on non-windows systems and this was my attempt to fix them.